napari/utils/info.py

```diff
@@ -69,6 +71,31 @@ def _sys_name():
     return ""
 
 
+class GlInfo(NamedTuple):
+    version: str
+    max_texture: int
+    extensions: str
+
+
+@lru_cache(maxsize=1)
```

Comment:
Thoughts on putting all this in our `gl.py` file in our utils folder in `_vispy`? It feels like it would live there much more naturally? https://github.com/napari/napari/blob/main/napari/_vispy/utils/gl.py

Reply:
agree!  will move